src/main/java/frc/robot/subsystems/vision/NoteCamera.java

```diff
@@ -0,0 +1,50 @@
+package frc.robot.subsystems.vision;
+
+
+import org.littletonrobotics.junction.Logger;
+import org.photonvision.PhotonCamera;
+import org.photonvision.targeting.PhotonTrackedTarget;
+
+public class NoteCamera {
+
+  PhotonCamera camera;
+  String path;
+
+  private final double MIN_AREA = 10;
+
+  public NoteCamera(String name) {
+    // This is a constructor
+    camera = new PhotonCamera(name);
+    path = "Vision/" + name.replace("_", "");
+  }
+
+  public PhotonTrackedTarget getTarget() {
+    if(1==1) return null;
```

Comment:
This looks like it will always evaluate to true, what's the purpose of this?

Reply:
to disable because its crashing in a loop when you have two minutes left until the next practice match and the java compiler throws errors on unreachable code :(

(it's on the agenda to debug, just low prio rn since the actual aligning code is scuffed)